src/utils/stringFormat.tsx

```diff
@@ -158,3 +162,24 @@ export function parseLatLng(str: string): Coordinate | null {
 
   return [lat, lng];
 }
+
+export function generateTreeCareMessage(item: Activity): string {
+  const activityStrings = [] as string[];
+  if (item.cleaned) activityStrings.push('cleared of waste');
+  if (item.mulched) activityStrings.push('mulched');
+  if (item.watered) activityStrings.push('watered');
+  if (item.weeded) activityStrings.push('weeded');
+
+  const numberOfActivities = activityStrings.length;
+
+  // invariant: at least one activity will be present
```

Comment:
⛏️ Just to be safe, could we add this here? Because technically, there's nothing stopping someone from passing in all `false`s

```
if (numberOfActivities === 0) {
  throw new Error('At least one activity must be true');
}
```

Reply:
`Intl.ListFormat` seems really cool! Would love to use it, but looks like it's really [only supported by browser versions from 2020/1 onwards](https://caniuse.com/?search=Intl.ListFormat) - is that something we really worry about?

Reply:
I think for our case, 91% global support is good enough. For reference, syntax like the [optional chaining operator](https://caniuse.com/mdn-javascript_operators_optional_chaining) and the [nullish coalescing operator](https://caniuse.com/mdn-javascript_operators_nullish_coalescing) (which we do use sporadically) have similar browser support.